packages/frontend-2/pages/server-management/index.vue

```diff
@@ -0,0 +1,210 @@
+import Card, { CardInfo } from '../../components/server-management/Card.vue'
+import SettingsDialog from '../../components/server-management/SettingsDialog.vue'
+import {
+  ServerIcon,
+  UsersIcon,
+  EnvelopeIcon,
+  ChartBarIcon,
+  HomeIcon
+} from '@heroicons/vue/24/solid'
+import { useRouter } from 'vue-router'
+
+interface GithubRelease {
+  url: string
+  assets_url: string
+  upload_url: string
+  html_url: string
+  id: number
+  node_id: string
+  tag_name: string
+}
+
+interface SettingsDialogRef {
+  onSubmit: () => void
+}
+
+const router = useRouter()
+const showDialog = ref(false)
+const settingsDialog = ref<SettingsDialogRef | null>(null)
+const latestVersion = ref<string | null>(null)
+const isLatestVersion = ref<boolean>(false)
+
+const dataQuery = graphql(`
+  query Admin {
+    admin {
+      serverStatistics {
+        totalPendingInvites
+        totalProjectCount
+        totalUserCount
+      }
+      inviteList {
+        totalCount
+      }
+    }
+    serverInfo {
+      name
+      version
+    }
+  }
+`)
+
+const { result, refetch } = useQuery(dataQuery)
+const versionInfo = reactive({
+  current: result.value?.serverInfo.version || 'N/A',
+  latest: latestVersion.value || 'N/A'
+})
+
+const serverData = computed((): CardInfo[] => [
+  {
+    title: 'Server Name',
+    value: result.value?.serverInfo.name || 'N/A',
+    icon: ServerIcon,
+    cta: {
+      type: 'button',
+      label: 'Edit Settings',
+      action: () => {
+        showDialog.value = true
+      }
+    }
+  },
+  {
+    title: 'Speckle Version',
+    value: versionInfo.current,
+    icon: ChartBarIcon,
+    cta: !isLatestVersion.value
+      ? {
+          type: 'link',
+          label: 'Update is available',
+          action: () => {
+            window.open(
+              'https://github.com/specklesystems/speckle-server/releases',
+              '_blank'
+            )
+          }
+        }
+      : undefined
+  }
+])
+const userData = computed(() => [
+  {
+    title: 'Active users',
+    value: result.value?.admin.serverStatistics.totalUserCount?.toString() || 'N/A',
+    icon: UsersIcon,
+    cta: {
+      type: 'button',
+      label: 'Manage',
+      action: async () => await router.push('/server-management/active-users/')
+    }
+  },
+  {
+    title: 'Pending invitations',
+    value: result.value?.admin.inviteList.totalCount?.toString() || 'N/A',
+    icon: EnvelopeIcon,
+    cta: {
+      type: 'button',
+      label: 'Manage',
+      action: async () => await router.push('/server-management/pending-invitations/')
+    }
+  }
+])
+const projectData = computed(() => [
+  {
+    title: 'Projects',
+    value: result.value?.admin.serverStatistics.totalProjectCount?.toString() || 'N/A',
+    icon: HomeIcon,
+    cta: {
+      type: 'button',
+      label: 'Manage',
+      action: async () => await router.push('/server-management/projects/')
+    }
+  }
+])
+
+onMounted(async () => {
```

Comment:
1. Don't do this in `onMounted`, its going to cause hydration mismatch warnings.
2. You don't need to manually trigger `fetchData()` the initial load for queries happens automatically